SPECS/kubevirt/kubevirt.spec

```diff
@@ -18,8 +18,8 @@
 
 Summary:        Container native virtualization
 Name:           kubevirt
-Version:        0.59.0
-Release:        14%{?dist}
+Version:        1.2.0
```

Comment:
just double checking, there are no changes made to the spec other than version??  that seems rare.